toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/Scalebar.kt

```diff
@@ -143,6 +143,8 @@ public fun Scalebar(
             isScalebarVisible.value = true
             delay(autoHideDelay)
             isScalebarVisible.value = false
+        } else {
```

Comment:
Or alternatively, we could set `isScalebarVisible` to true only when the duration is `INFINITE` (Do nothing when `ZERO`?) 
```suggestion
        } else if (autoHideDelay == Duration.INFINITE) {
```